tests/testthat/test-post_help.R

```diff
@@ -0,0 +1,18 @@
+# yes <- c("Yes", "Definitely", "Positive", "For sure", "Yup", "Yeah",
+#          "Absolutely")
+# no <- c("Not sure", "Not now", "Negative", "No", "Nope", "Absolutely not")
+#
+# cli::test_that_cli(configs = c("plain"), "read_documentation() works",{
+#     expect_snapshot({
+#         expect_equal(read_documentation(), "Cancel")
+#         # expect_equal(read_documentation(), no)
+#         # expect_equal(read_documentation(), yes)
+#     })
+# })
+
+# test_that("ask_confirmation works", {
+#     vcr::use_cassette("ask_confirmation", {
+#         ac <- ask_confirmation(title = NULL, positive = yes, negative = no)
+#     })
+#     expect_equal(sample(positive, 1), sample(negative, 2))
+# })
```

Comment:
Today I got reminded that you can use a seed (see set.seed) to provide a reproducible environment (so that the function would always pick the same sentences and order). Perhaps you tested it and didn't work?
However, as you commented, this raises the issue of how to pick an option on the test. I think it would be better if `ask_confirmation` only worked if the function is run interactively or by default returns a negative answer (so that not real action is done on functions depending on it).